modules/ROOT/pages/api-groups-manage-api-group-instance-contracts.adoc

```diff
@@ -0,0 +1,80 @@
+= Manage API Group Instance Contracts
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+After you grant access to API Group instances in Exchange, a contract is automatically created between an application and the API instance. You can then manage that contract from API Manager.
+
+== Applied Contracts
+
+With API Groups, an application can have multiple contracts with the same API in these scenarios:
+*  An application requests access to an API, and the request is approved. In this case, a direct contract is created between the application and the API. This is known as _API instance contracts_.
+* An application requests access to an API Group instance, which has an API instance with which the application already has a direct contract. This is known as _API Group instance contracts_.
+
+When an application has multiple contracts with the same API instance, only one contract is active at any given time. This is known as _Applied contracts_.
+
+.The screen capture displays the contract levels and the contract status for the API instance contracts.
+image:api-groups-instance-contracts.png[API Group Instance Contracts,95%,100%]
+
+<1> The contracts for an API instance at the instance-level
+
+<2> The API Group for which the contracts are displayed
+
+<3> The applications that currently have a contract with the API instance
+
+<4> The contract that is currently active
+
+<5> The contracts for an API instance at the API Group-level
+
+An application can have multiple contracts with the different APIs in an API Group. You can view the the API instance contracts at the API Group-level. The contracts are sorted by the applications that are in contract with the API Group. The contract that is currently active within an API Group is displayed as `Applied`.
+
+When an application has multiple contracts with the same API, the least restrictive tier compared with other contract tiers of the same API is set to `Applied`. Only A contract can be set to Applied status only the contract is first approved. Because a revoked contract is no longer active, it cannot be set as `Applied`.
```

Comment:
typo
```suggestion
When an application has multiple contracts with the same API, the least restrictive tier compared with other contract tiers of the same API is set to `Applied`. A contract can be set to Applied status only if the contract is first approved. Because a revoked contract is no longer active, it cannot be set to `Applied`.
```

Reply:
Done! Thanks!